build/docker/debian/Dockerfile

```diff
@@ -157,7 +157,7 @@ ENV PATH /usr/local/go/bin:$PATH
 
 # Haxe
 RUN mkdir -p /usr/lib/haxe && \
-    curl http://haxe.org/website-content/downloads/3.2.0/downloads/haxe-3.2.0-linux64.tar.gz | \
+    wget -O - https://github.com/HaxeFoundation/haxe/releases/download/3.4.2/haxe-3.4.2-linux64.tar.gz | \
```

Comment:
Will wait for THRIFT-4141 and merge out the Dockerfile change from this pull request.